lib/rules/template-no-let-reference.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
       description: 'disallow referencing let variables in \\<template\\>',
       category: 'Ember Octane',
       recommended: true,
+      recommended_for: 'gjs',
```

Comment:
Can we follow the current pattern which is `[configName]: true` to indicate if the rule belongs to a config?

Reply:
You mean replace recommended_for with configName: 'gts'

Reply:
Any examples for that property?

Reply:
No, we should use the config name as the property name, no shorthand:

```
recommendedGts: true
```

And then we can automatically transform camelCase to kebab-case when checking if a rule is enabled for a config.